db/model/aspect.js

```diff
@@ -341,6 +353,28 @@ module.exports = function aspect(seq, dataTypes) {
             resolve(found.length === 1);
           }));
       }, // isWritableBy
+
+      checkGeneratorReferences(action) {
+        const aspectName = this.previous('name');
+        const where = { where: { aspects: { $contains: [aspectName] } } };
```

Comment:
"contains" is case sensitive so we may have to use a more expensive query to check for reference :(